### PR TITLE
little updates on installation

### PR DESCRIPTION
**Pull request by jalogisch**
Added some additional information on the configuration/setup

### CODE REVIEW

File: pages/installation/operating_system_packages.rst [hunk 1]
```diff
@@ -131,5 +131,6 @@ Step-by-step guides
 Feedback
 --------
 
-Please open an `issue <https://github.com/Graylog2/fpm-recipes/issues>`_ in the `Github repository <https://github.com/Graylog2/fpm-recipes>`_ if you
-run into any packaging related issues. **Thank you!**
+Please open an `issue in the Github repository for the operating system packages <https://github.com/Graylog2/fpm-recipes>`__ if you
+run into any packaging related issues. If this Documentation is confusing or left questions to you please open an `issue in the Github repository for the documentation <https://github.com/Graylog2/documentation/issues>`__.
```
Comment (joschi) [inline pages/installation/operating_system_packages.rst]: Lower case "documentation" (also in all following files of this change set).

Maybe reword to "If you found this documentation confusing or have more questions, please open an issue…"

File: pages/installation/os/centos.rst [hunk 2]
```diff
@@ -85,6 +85,9 @@ The last step is to enable Graylog during the operating system's startup::
   $ sudo systemctl start graylog-server.service
 
 
+.. note:: We higly recommend that you proxy the Graylog Webinterface and API with NGINX/Apache like described in: :ref:`configuring_webif_nginx`
```
Comment (joschi) [inline pages/installation/os/centos.rst]: Missing "." after the `ref`erence.

Also, why do we recommend this?

Comment (jalogisch) [inline pages/installation/os/centos.rst]: IMHO this Step-by-step is used by people that are not very skilled on linux and for them it is easier to run HTTPS from NGINX/Apache.

File: pages/installation/os/centos.rst [hunk 3]
```diff
@@ -85,6 +85,9 @@ The last step is to enable Graylog during the operating system's startup::
   $ sudo systemctl start graylog-server.service
 
 
+.. note:: We higly recommend that you proxy the Graylog Webinterface and API with NGINX/Apache like described in: :ref:`configuring_webif_nginx`.
```
Comment (joschi) [inline pages/installation/os/centos.rst]: I think we should remove this note and rather improve our own HTTPS documentation.

Otherwise we could as well remove HTTPS support from Graylog and always recommend using a 3rd party reverse proxy.

Comment (kroepke) [inline pages/installation/os/centos.rst]: same as my comment in https://github.com/Graylog2/documentation/pull/125#issuecomment-231743893
